src/stratis_cli/_errors.py

```diff
@@ -330,7 +330,9 @@ def __init__(self, command_line_args, namespace):
         self.command_line_args = command_line_args
         self.namespace = namespace
 
-    def __str__(self):
+    # pylint: disable=fixme
```

Comment:
Also, explain what needs to be fixed here...that coverage should be obtained, you should use the same formula as the others.